### DISCUSSION TITLE
Can't find a way to start or stop recording on Open GoPro. Im I missing something? Is it possible?

### DISCUSSION THREAD

**Discussion by ghost** [gopro/OpenGoPro#623] [Q&A] [answered]
I've been reading the Open GoPro and Labs documentation, but I can't find a simple way to take photos or start recording. Is this intentional, or am I just missing something?

Thanks in advance.

**Comment by cloudchef-wyatt** [comment]
1. Define your use case. Are you controlling the camera through the mobile app?
2. Define what you mean by "simple". Simple is subjective and depends on the context in which you intend to use the product.
3. This repo contains plenty of demos, I would recommend starting there.

**Comment by kmatzen** [comment, marked as answer]
You're looking for the "set shutter" command.  
https://gopro.github.io/OpenGoPro/ble/features/control.html#set-shutter
https://gopro.github.io/OpenGoPro/http#tag/Control/operation/OGP_SHUTTER
